File: java/no-package/basic/RectangleArea.java
import java.util.Scanner;

public class RectangleArea {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Enter length: ");
        float l = s.nextFloat();
        System.out.print("Enter breadth: ");
        float b = s.nextFloat();
        float a = l * b;
        System.out.println("Area is " + a);
    }
}

File: c/basic/rectanglearea.c
#include <stdio.h>

int main(void)            // f(n) = 1 => f(1) => O(1)
{
    float length, breadth, area;

    printf("Enter the length of the rectangle: ");
    scanf("%f", &length);
    printf("Enter the breadth of the rectangle: ");
    scanf("%f", &breadth);

    area = length * breadth;

    printf("The area of the rectangle is: %.2f\n", area);

    return 0;
}